primitives/core/src/crypto.rs

```diff
@@ -18,7 +18,9 @@
 //! Cryptographic utilities.
 // end::description[]
 
-use rstd::{vec::Vec, hash::Hash};
+#[cfg(feature = "std")]
+use rstd::vec::Vec;
```

Comment:
`Vec` is in `std::prelude`. You probably want to use the `full-crypto` feature here.

Reply:
You really need to use the full-crypto feature here.